sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/ComplexTypes.scala

```diff
@@ -19,57 +19,47 @@ package org.apache.spark.sql.catalyst.optimizer
 
 import org.apache.spark.sql.catalyst.expressions._
 import org.apache.spark.sql.catalyst.plans.logical.LogicalPlan
+import org.apache.spark.sql.catalyst.plans.logical.Aggregate

```

Comment:
This should be before line 21 in alphabetical order.
You can check this locally with `dev/scalastyle`.

Reply:
> import org.apache.spark.sql.catalyst.plans.logical.{Aggregate, LogicalPlan}